packages/eslint-config-react-native/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "eslint-config-react-native",
+  "description": "STRV's ESLint config for React Native projects",
+  "version": "1.0.0",
+  "author": "André Nanninga <andre@nannin.ga>",
+  "bugs": {
+    "url": "https://github.com/strvcom/code-quality-tools/issues"
+  },
+  "keywords": [
+    "config",
+    "eslint",
+    "react-native",
+    "strv"
+  ],
+  "license": "BSD-3-Clause",
+  "main": "index.js",
+  "directories": {
+    "lib": "lib",
+    "test": "__tests__"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/strvcom/code-quality-tools.git"
```

Comment:
Not sure if this is important but in the other _package.json_ files we only have the `git://` protocol mentioned in the url. Don't know what's the difference, honestly. 🤷‍♂️